app/services/bulk_upload/lettings/year2022/row_parser.rb

```diff
@@ -770,7 +770,7 @@ def validate_incomplete_soft_validations
       next if question.completed?(log)
 
       question.page.interruption_screen_question_ids.each do |interruption_screen_question_id|
-        field_mapping_for_errors[interruption_screen_question_id.to_sym].each do |field|
+        field_mapping_for_errors[interruption_screen_question_id.to_sym]&.each do |field|
```

Comment:
Are you sure this is what you want to do? `nil#each` will still error. How about `field_mapping_for_errors.fetch(interruption_screen_question_id.to_sym, [])`?